routes/users.js

```diff
@@ -1,9 +1,37 @@
 var express = require('express');
 var router = express.Router();
+var User = require('./../models').User;
+
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
+const uuidv4 = require('uuid/v4');
+
 
 /* GET users listing. */
-router.get('/', function(req, res, next) {
-  res.send('respond with a resource');
+router.post('/', function(req, res, next) {
+  if (req.body.email && req.body.password && req.body.passwordConfirmation) {
+    if (req.body.password == req.body.passwordConfirmation) {
+      User.create({
+        email: req.body.email,
+        password: bcrypt.hashSync(req.body.password, saltRounds),
+        apiKey: uuidv4()
+      })
+      .then(user =>{
+        res.setHeader("Content-type", "application/json");
```

Comment:
Hey! I believe you can move this line (setHeader) to the top of the request (under line 11) and dry up your code.

Reply:
YES! I wasn't sure exactly where to place this since I was using a couple of if statements first. Will try to figure this out